src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -261,12 +261,16 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
         }
 
         private static bool BackendSupportsCustomCertificateHandling =>
-            RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
+#if unix
```

Comment:
This code was in the Unix only compile of the tests you added. Without the 'if', it will end up p/invoking into a unix API.  Won't that fail?